spring-cloud-azure-servicebus-stream-binder/src/test/java/com/microsoft/azure/servicebus/stream/binder/ServiceBusPartitionBinderTests.java

```diff
@@ -0,0 +1,154 @@
+        properties.setHeaderMode(HeaderMode.embeddedHeaders);
+        return properties;
+    }
+
+    @Override
+    public Spy spyOn(String name) {
+        return null;
+    }
+
+    @Override
+    public void testClean() throws Exception {
+        // No-op
+    }
+
+    @Override
+    public void testOneRequiredGroup() {
+        // Required group test rely on unimplemented start position of consumer properties
+    }
+
+    @Override
+    public void testTwoRequiredGroups() {
+        // Required group test rely on unimplemented start position of consumer properties
+    }
+
+    @Override
+    public void testPartitionedModuleJava() {
+        // Partitioned consumer mode unsupported yet
+        // since event hub processor can't only process certain partition
+    }
+
+    @Override
+    public void testPartitionedModuleSpEL() {
+        // Partitioned consumer mode unsupported
+        // since event hub processor can't only process certain partition
+    }
+
+    // Same logic as super.testSendAndReceiveNoOriginalContentType() except one line commented below
+    @Override
+    public void testSendAndReceiveNoOriginalContentType() throws Exception {
+        Binder binder = this.getBinder();
+        BindingProperties producerBindingProperties =
+                this.createProducerBindingProperties(this.createProducerProperties());
+        DirectChannel moduleOutputChannel = this.createBindableChannel("output", producerBindingProperties);
+        BindingProperties inputBindingProperties =
+                this.createConsumerBindingProperties(this.createConsumerProperties());
+        DirectChannel moduleInputChannel = this.createBindableChannel("input", inputBindingProperties);
+        Binding<MessageChannel> producerBinding =
+                binder.bindProducer(String.format("bar%s0", this.getDestinationNameDelimiter()), moduleOutputChannel,
+                        producerBindingProperties.getProducer());
+        Binding<MessageChannel> consumerBinding =
+                binder.bindConsumer(String.format("bar%s0", this.getDestinationNameDelimiter()),
+                        "testSendAndReceiveNoOriginalContentType", moduleInputChannel, this.createConsumerProperties());
+        this.binderBindUnbindLatency();
+        Message<?> message =
+                MessageBuilder.withPayload("foo").setHeader("contentType", MimeTypeUtils.TEXT_PLAIN).build();
+        // Comment line below since event hub operation is event driven mode
+        // but subscriber is not ready in the downstream
+        //moduleOutputChannel.send(message);
+
+        final CountDownLatch latch = new CountDownLatch(1);
+        final AtomicReference<Message<byte[]>> inboundMessageRef = new AtomicReference();
+        moduleInputChannel.subscribe(message1 -> {
+            try {
+                inboundMessageRef.set((Message<byte[]>) message1);
+            } finally {
+                latch.countDown();
+            }
+
```

Comment:
better to move this blank line before line 143.

Reply:
Yes